### ISSUE TITLE
Refine the README.md

### ISSUE THREAD

**Issue by kemaswill** [farizrahman4u/keras-contrib#4]
I think we'd better demonstrate the purpose of this new keras-contrib repository in the README.md. From my point of view, the purpose is to:

1. provide an official keras extension repository for the keras community, which contains newest layers, activations, loss functions, optimizers, and so on.
2. keep the original keras repository clean, only the most valuable functions will be finally merged into the original keras repository.

Any suggestions?

**Comment by farizrahman4u** [comment]
@patyork
